docs/backend/guides/how-to/upgrade-go-version.md

```diff
@@ -67,15 +67,13 @@ go version go1.16.4 darwin/amd64
     - `Dockerfile.webhook_client`
     - `Dockerfile.webhook_client_dp3`
     - `Dockerfile.webhook_client_local`
-    - `scripts/run-e2e-mtls-test-docker`
-    - `scripts/run-e2e-test-docker`
     - `Makefile` (in the `docker_circleci` target)
 - If you use `asdf` to manage your local Go version, you still need to update the nix package in `nix/default.nix` - use the [package search](https://ahobson.github.io/nix-package-search) to find the hash
 - If the major/minor version changed (the first or second number, e.g. 1.x.y to 2.x.y or 1.15.x to 1.16.x):
-  - [Example PR](https://github.com/transcom/mymove/pull/4990)
+  - [Example PR](https://github.com/transcom/mymove/pull/10185)
   - Update the following files with the new go version:
-    - `go-version` in `.github/workflows/go-auto-approve.yml`
     - `go.mod`
+  - You may have to update the `golangci-lint` pre-commit hook version found in `.pre-commit-config.yaml` to one that supports the new Go version
 - Rerun the Go formatter on the codebase with `pre-commit run --all-files golangci-lint`
 - Regenerate mocks (in case the signatures have changed that we're mocking): `make mocks_generate`
 - Run `make e2e_test_docker` to test that the `Dockerfile.*local` files work  with the new image.
```

Comment:
I suppose for maximum safety, we should always deploy to experimental for go upgrades, but is that too much? I guess the only time we ever had a problem was when the base image changed, not the go version, right?

Reply:
I might be forgetting something, but I don't think I've ever seen a problem directly attributable to a go upgrade that would have been caught by an experimental deploy.  The wildcard is that you might pick up some other (perhaps untested) merges to the main circleci-docker branch like we did with the base image.  I think I'll add in a warning about that here.